packages/liquidator/index.js

```diff
@@ -209,7 +221,10 @@ async function Poll(callback) {
     // This object is spread when calling the `run` function below. It relies on the object enumeration order and must
     // match the order of parameters defined in the`run` function.
     const executionParameters = {
+      // EMP Address. Should be an Ethereum address
       empAddress: process.env.EMP_ADDRESS,
+      // One Split address. Should be an Ethereum address. Defaults to mainnet address 1split.eth
+      oneSplitAddress: process.env.ONE_SPLIT_ADDRESS || ONE_SPLIT_ADDRESS,
```

Comment:
I think you should remove the `|| ONE_SPLIT_ADDRESS`. This way we can choose to not configure the 1inch integration